transform/kubernetes/kubernetes.go

```diff
@@ -13,10 +13,12 @@ import (
 	"github.com/konveyor/crane-lib/transform/util"
 	"github.com/konveyor/crane-lib/version"
 	ocpappsv1 "github.com/openshift/api/apps/v1"
+	ocprbacv1 "github.com/openshift/api/authorization/v1"
```

Comment:
Shouldn't the ocp role binding bits be in the openshift plugin? The kubernetes plugin should be limited to core kube resources, since the OpenShift plugin won't be installed or executed for non-OpenShift clusters.

Reply:
I've removed the openshift rolebinding stuff from here and I'll add it to the plugin shortly. I'll follow up with PRs to move the pvc remapping for deploymentConfigs.

Reply:
I want to re open this, on any new version of openshift only k8s bracket is needed. We should just convert everything to this and only use this IMO

Reply:
rbac sorry phone

Reply:
we would need to force `crane export` and the discovery piece to force apiServer to give the k8s rbac. At least, that is how I understand it from what @jmontleon demonstrated (with their being a 1-to-1 relationship of k8s rbac and os rbac) in addition to `crane export` only showing the os rbac.